client/types/account-overview.d.ts

```diff
@@ -25,6 +25,10 @@ export interface Account {
 		 */
 		monthly_anchor: number;
 	};
+	default_external_accounts: {
+		currency: string;
+		status: string;
+	}[];
```

Comment:
💭 it may be helpful to add some descriptions here.
```suggestion
	/**
	 * Default deposit destination external accounts (max one per currency).
	 */
	default_external_accounts: {
		/**
		 * The currency of the external account. e.g. eur, chf.
		 */
		currency: string;
		/**
		 * The status of the external account. e.g. new, errored.
		 */
		status: string;
	}[];
```